integration-test/test.bats

```diff
@@ -101,22 +130,35 @@
 
 @test "govvv build - reads Version from ./VERSION file" {
     tmp="${BATS_TMPDIR}/a.out"
-    run bash -c "cd ${BATS_TEST_DIRNAME}/app-versioned && govvv build -o ${tmp} ."
+    run bash -c "cd ${BATS_TEST_DIRNAME}/app-versioned && govvv build -o ${tmp}"
     echo "$output"
     [ "$status" -eq 0 ]
 
     run "$tmp"
     echo "$output"
     [ "$status" -eq 0 ]
-    [[ "$output" == "Version=2.0.1" ]]
+    [[ "$output" == "Version=2.0.1-app-versioned" ]]
+}
+
+@test "govvv build - ./VERSION file overridden by -version option" {
+    tmp="${BATS_TMPDIR}/a.out"
+    run bash -c "cd ${BATS_TEST_DIRNAME}/app-versioned && govvv build -o ${tmp} -version 1.2.3-command-line"
+    echo "$output"
+    [ "$status" -eq 0 ]
+
+    run "$tmp"
+    echo "$output"
+    [ "$status" -eq 0 ]
+    [[ "$output" == "Version=1.2.3-command-line" ]]
 }
 
 @test "govvv compiled with govvv" {
-    run govvv install -a
+    run govvv build
     echo "$output"
     [ "$status" -eq 0 ]
 
-    run govvv
+    run ./govvv
     echo "$output"
     [[ "${lines[1]}" =~ ^version:\ (.*)@[0-9a-f]{4,15}-(dirty|clean)$ ]]
+	rm ./govvv
```

Comment:
similarly, try to stick to the existing setup. it's idiomatic and works.

Reply:
See above replies.